index.html

```diff
@@ -22,17 +22,329 @@
+            </ul>
+        </nav>
+    
+        <nav class="container44">
+            <ul class="navlink7">
+                <li class="project6"><a href="#">See this Project</a></li>
+                <li class="arrow5"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+
+
+
+    <div class="container45">
+        <h3>About</h3>
+    </div>
+    <div class="container46">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </div>
+    <div class="container47">
+        <div class="CV">
+            <p>Get my CV</p>
         </div>
+    </div>
+    <nav>
+        <ul>
+            <li class="arrow6"><a href="#"><img src="imgformobile/Rectangle45.png" alt="sign"></a></li>
+            <li class="arrow7"><a href="#"><img src="imgformobile/Union1.png" alt="sign"></a></li>
+        </ul>
+    </nav>   
+    
+    
+    <nav class="container48">
+        <ul class="box1">
+            <li><a href ="#">Languages</a></li>
+        </ul>
+    </nav>
+        <nav class="Language">
+            <ul>
+                <li><a href ="#">JavaScript</a></li>
+                <li><a href ="#">Ruby</a></li>
+                <li><a href ="#">HTML</a></li>
+                <li><a href ="#">CSS</a></li>
+            </ul>
+        </nav>
+    <div class="container51">
+        <img src="imgformobile/hash.png" alt="shape">
+    </div>
+    
+    <nav class="container49">
+        <ul class="box2">
+            <li><a href ="#">Frameworks</a></li>
+        </ul>
+    </nav>
+        <nav class="Frame">
+            <ul>
+                <li><a href ="#">Bootstrap</a></li>
+                <li><a href ="#">Ruby on Rails</a></li>
+                <li><a href ="#">RSpec</a></li>
+                <li><a href ="#">Capibara</a></li>
+                <li><a href ="#">Selenium</a></li> 
+            </ul>
+        </nav>
+        
+        <nav class="container52">
+            <ul class="star">
+                <li class="shape18"><img src="imgformobile/Rectangle18.png" alt="shape"></li>
+                <li class="shape19"><img src="imgformobile/Rectangle19.png" alt="shape"></li>
+            </ul>
+        </nav>
+
+    <nav class="container50">
+        <ul class="box3">
+        <li><a href="#">Skills</a></li>
+        </ul>
+    </nav>
+    <nav>
+        <ul class="MySkill">
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+        </ul>    
+    </nav>
+    
+    <nav class="container53">
+        <ul class="bulb">
+            <li class="shape20"><img src="imgformobile/Vector.png" alt="shape"></li>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_317](https://user-images.githubusercontent.com/69638013/177507226-1a6d664d-3636-4854-a4b0-618810b4fed0.png) | ![Screenshot_316](https://user-images.githubusercontent.com/69638013/177507205-36c31c35-8c76-4ae3-a651-fc03deb9747b.png)


- [ ] Kindly align icon to the right and delete all images not used in your design 👍🏿